### PR TITLE
Implements Granodd's resprites for the Dominian voidsuit and RIG, also adds an Unathi version of each

### PR DESCRIPTION
**Pull request by Jaraci**
See title.

image

Voidsuits are on the left, RIGs are on the right.

All credit to @granodd for the spritework here -- I'm just coding the implementation for them.

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

**Review by alsoandanswer — COMMENTED** [review]
This is pretty good, but since the Unathi version doesn't have any mechanical differences, the regular voidsuit sprite should just change with the species.

An example of this is how Vaurcae bulwarks wear uniforms.

If you do this change I'll approve this.

File: html/changelogs/omicega-granodddominiastuff.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds Granodd's resprites for the Dominian voidsuit and RIGsuit, which now include an Unathi variant for each.
```
Comment (SleepyGemmy) [inline html/changelogs/omicega-granodddominiastuff.yml:13]:
```suggestion
  - rscadd: "Adds Granodd's resprites for the Dominian voidsuit and RIGsuit, which now include an Unathi variant for each."
```

**Comment by Jaraci** [comment]
> This is pretty good, but since the Unathi version doesn't have any mechanical differences, the regular voidsuit sprite should just change with the species.

This isn't going to preserve the item sprites for the Unathi voidsuit, though, and as far as I'm aware no voidsuits work like this to date -- they all have to be manually refitted to the new species' shape. I'm not aware of any generic suit cyclers available to antags like traitors or mercenaries who might be making use of this suit, though, so implementing them as separate voidsuits was the best option I could see.

**Review by alsoandanswer — APPROVED** [review]
Good lord, that is terrible. RIGs should be updated to use the new code in the near future, honestly.

Sorry about the inconvenience.

**Review by Alberyk — APPROVED** [review]

File: html/changelogs/omicega-granodddominiastuff.yml (hunk 1 above)
Comment (NonQueueingMatt) [inline html/changelogs/omicega-granodddominiastuff.yml:13]:
```suggestion
  - rscadd: "Adds Granodd's resprites for the Dominian voidsuit and RIGsuit, which now include an Unathi variant for each."
```

**Pull request merged**